app/controllers/service_requests_controller.rb

```diff
@@ -437,9 +437,9 @@ def remove_service
     to_delete = @service_request.sub_service_requests.map(&:organization_id) - @service_request.service_list.keys
     to_delete.each do |org_id|
       ssr = @service_request.sub_service_requests.find_by_organization_id(org_id)
-      if !['first_draft', 'draft'].include?(@service_request.status) and !@service_request.submitted_at.nil? and @service_request.submitted_at > ssr.created_at
+      if @service_request.line_items.count.zero? && !ssr.submitted_at.nil?
```

Comment:
This a pretty hefty change. Can you explain why we're now only looking at line items count?